book/03-git-branching/sections/workflows.asc

```diff
@@ -23,7 +23,7 @@ It's generally easier to think about them as work silos, where sets of commits g
 
 [[lrbranch_b]]
 .A "`silo`" view of progressive-stability branching
-image::images/lr-branches-2.png[A "`silo`" view of progressive-stability branching]
+image::images/lr-branches-2.png[A “silo” view of progressive-stability branching']
```

Comment:
This is weird. It recommended you freeze the curly quotes? I'd rather they just be straight quotes if we can't use the ``` "`backtick`" ``` syntax we use elsewhere.

Also, what's with the trailing single quote?

Reply:
I don't know if it's a feature or a bug of asciidoctor, but the backticks macro is not converted in the `alt` attributes of images, leading to a breakage of xhtml syntax.

Single quoting may work and remain as is, but it's a semantic change. And anyway, is an alternate text that just replicates the caption really needed?

Reply:
For ref, https://github.com/asciidoctor/asciidoctor/issues/3864

Reply:
OK, for the trailing single quote.

Would you prefer converting:

* “ → '
* ” → ' 

in alternative texts?

Reply:
Yeah, this is a fair point. Ideally we'd have semantic alt-text that actually helps in a sight-impaired scenario. I don't remember why we did it this way.

I guess it's fine, though.